compositor_integration_tests/src/tests.rs

```diff
@@ -0,0 +1,13 @@
+use anyhow::Result;
+
+use crate::CompositorInstance;
+
+use self::simple::run_simple_test;
+
+mod simple;
+
+pub fn integration_tests(update_dumps: bool) -> Result<()> {
+    run_simple_test(CompositorInstance::start(8000), update_dumps)?;
```

Comment:
I assume all tests are expected to be added to this function. Do we need to enforce running all the test sequentially? Can we maybe support  running them concurrentlly?

Reply:
If we can run them concurrentlly, then I would expect each test to be separate function marked with `#[test]`, but I'm not sure how updating tests would work in that scenario. (Maybe it could be done with feature)

Reply:
I did consider marking each test with `#[test]`, but as you said, updating those tests was problematic